src/libaudqt/info-widget.cc

```diff
@@ -55,6 +59,10 @@ static const TupleFieldMap tuple_field_map[] = {
     {N_("Codec"), Tuple::Codec, false},
     {N_("Quality"), Tuple::Quality, false},
     {N_("Bitrate"), Tuple::Bitrate, false},
+
+    {nullptr, Tuple::Invalid, false},
+    {N_("Folder"), Tuple::Path, false},
+    {N_("Filename"), Tuple::Basename, false},
```

Comment:
I added a label showing the URI below the album art as we do in GTKUI, so this is redundant now.